corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -136,7 +136,7 @@ def extend(self, iterable):
 
     def _write_to_file(self, fileobj):
         # Add 1 to num_items to account for message element
-        items = (self.items_template % bytes(self.num_items + 1)) if self.items else b''
+        items = (self.items_template % ('%s' % (self.num_items + 1)).encode('utf-8')) if self.items else b''
```

Comment:
This whole class should handle unicode instead of bytes, but that's out of scope for this PR.